docs/source/examples/basic_usage.rst

```diff
@@ -1,12 +1,14 @@
 Basic Usage
 ===========
 
+This example shows how to use TorchJD to perform an iteration of Jacobian descent on a regression
+model. In this example, a batch of inputs is forwarded through the model and two corresponding
```

Comment:
I think that before explain exactly what happens at a low level (as in the 2nd sentence), we need to explain a bit more what we're trying to do: train a regression model with 2 objectives.

I would simply add "with two objectives" right at the end of the first sentence.